### PR TITLE
Add taxonomy examples

### PR DESCRIPTION
**Pull request by mavam**
###  :notebook_with_decorative_cover: Description

This PR adds some example concepts for VAST's builtin canonical taxonmy.

###  :memo: Checklist

- [x] Discuss how we ship our taxonomy declarations and definitions
- [x] All user-facing changes have changelog entries.
- [ ] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

File-by-file feels best.

### CODE REVIEW

File: schema/sysmon.yaml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      - sysmon.ProcessAccess.TargetProcessId
+      - sysmon.ProcessChangedFileCreationTime.ProcessId
+      - sysmon.ProcessCreation.ProcessId
+      - sysmon.ProcessTerminated.ProcessId
+      - sysmon.RawAccessRead.ProcessId
+      - sysmon.RegistryEventKeyAndValueRename.ProcessId
+      - sysmon.RegistryEventObjectCreateAndDelete.ProcessId
+      - sysmon.RegistryEventValueSet.ProcessId
+- concept:
+    name: proc.name
```
Comment (tobim) [inline schema/sysmon.yaml:44]:
```suggestion
    name: proc.name
    fields:
```

File: schema/sysmon.yaml [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    name: proc.parent.name
+    fields:
+      - sysmon.ProcessCreation.ParentImage
+- concept:
+    name: proc.thread.id
+    fields:
+      - sysmon.CreateRemoteThread.NewThreadId
+      - sysmon.ProcessAccess.SourceThreadId
+- concept:
+    name: file.name
```
Comment (tobim) [inline schema/sysmon.yaml:101]:
```suggestion
    name: file.name
    fields:
```

**Review by tobim — CHANGES_REQUESTED** [review]
This is very much needed.

I think we should change the `load` handler in the type-registry to recurse into subdirectories. Let's do that in a separate PR though.

**Review by meta-cretin — COMMENTED** [review]
looks good

**Review by tobim — APPROVED** [review]

File: schema/taxonomy/ecs/source.yaml [hunk 3]
```diff
@@ -1,7 +1,8 @@
+# This is an example taxonomy to demonstrate how one could go after mapping
+# different concrete tools to ECS.
```
Comment (tobim) [inline schema/taxonomy/ecs/source.yaml:2]: As long as we don't recursivly taxonomies from subfolders, this won't get picked up. Therefore the failing integration test. Let's make this change in a later PR.

**Review by tobim — CHANGES_REQUESTED** [review]

File: schema/taxonomy/ecs/source.yaml (hunk 3 above)
Comment (mavam) [reply schema/taxonomy/ecs/source.yaml:2]: I've added a story for this.

**Review by tobim — APPROVED** [review]
Tested additionally with some internal sysmon data. Did not attempt to verify the mappings.

**Pull request merged**